types/components/FormLabel.d.ts

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
 
 import { BsPrefixComponent } from './helpers';
+import { ColProps } from './Col';
 
-export interface FormLabelProps {
+export interface FormLabelProps extends ColProps {
```

Comment:
I wonder if the right way to do this is via a type Union, e.g. `FormLabel | FormLabelWithColProps` there `column` is `true` or `false` specifically (instead of `boolean`) to allow TS to differentiate

Reply:
I started to do that, but didn't see that throughout the other typings so I wasn't sure if there was a preferred style. Let me write it up with the type Union.